build/gitlab-dc.common.yml

```diff
@@ -0,0 +1,57 @@
+version: '2.4'
```

Comment:
I would like to replace the docker-compose.common.yml with this file, but I do want to make small changes and not break too much at once.